perf/h.js

```diff
@@ -0,0 +1,26 @@
+import bench from 'nanobench'
+import { h } from '../src/h'
+
+bench("vnode with no child 1M times", b => {
+  b.start()
+  for (let i = 0; i < 10000000; i++) {
```

Comment:
Not at the moment :P

Just make sure to test the benches with the callbacks being called in the for loops vs. not abstracting the for loops. Curious to see if that would change the performance..